app/bundles/CoreBundle/Form/Type/EntityLookupType.php

```diff
@@ -62,7 +62,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     public function configureOptions(OptionsResolver $resolver): void
     {
         $resolver->setRequired(['model', 'ajax_lookup_action']);
-        $resolver->setDefined(['model_lookup_method', 'repo_lookup_method', 'lookup_arguments', 'model_key']);
+        $resolver->setDefined(['model_lookup_method', 'repo_lookup_method', 'lookup_arguments', 'model_key', 'lookup_method']);
```

Comment:
I actually made `lookup_method` variable to pass the information for which method to use. But, now after looking at it again, it can be done by `model_lookup_method` itself. I'll make the change 👍

Thank you!